plone/app/upgrade/v60/final.py

```diff
@@ -206,3 +206,23 @@ def fix_syndication_settings(context):
         key = f"{old_iface}.{fieldname}"
         if key in record_keys:
             del registry.records[key]
+
+
+def fix_tinymce_format_iconnames(context):
+    """Fix 'strike-through' and 'sourcecode'.
+
+    See https://github.com/plone/Products.CMFPlone/issues/3905
+    """
+    registry = getUtility(IRegistry)
+    record = registry.records.get("plone.inline_styles")
+    if record is None:
+        return
+    values = record.value
+    replacements = [
+        ("Superscript|superscript|superscript", "Superscript|superscript|super-script"),
```

Comment:
This is fixing the wrong line.  Afterwards, the control panel has this:

```
Strikethrough|strikethrough|strikethrough
Superscript|superscript|super-script
```

Reply:
Ouch sorry. Will fix it asap.

Reply:
fixed